docs/getting-started.md

```diff
@@ -240,7 +240,10 @@ or run Firecracker as `root` (via `sudo`).
 You can check your KVM setup with:
 
 ```bash
-[ -r /dev/kvm ] && [ -w /dev/kvm ] && [ $(uname -r) \> 4.14 ] && echo "OK" || echo "FAIL"
+[ -r /dev/kvm ] && [ -w /dev/kvm ]           \
+  && [ "$(uname -r | cut -d. -f1)" -ge 4 ]   \
+  && [ "$(uname -r | cut -d. -f2)" -ge 14 ]  \
+  && echo "OK" || echo "FAIL"
```

Comment:
I think the kernel version check warrants a personalized error message, so that in case of failure, users van be able to differentiate between `/dev/kvm` issues and old kernel versions